common-service-core/src/main/resources/core-log-messages.yml

```diff
@@ -176,6 +176,13 @@ messages:
       reason: "An invalid apiInfo parameter was found while parsing the service metadata."
       action: "Remove or fix the referenced metadata parameter."
 
+    - key: org.zowe.apiml.common.codeSnippetParsingError
```

Comment:
This message can be removed as it is not used.

Reply:
Removed